src/nodes/Node.ts

```diff
@@ -7,7 +7,11 @@ import type { Scalar } from './Scalar.js'
 import type { YAMLMap } from './YAMLMap.js'
 import type { YAMLSeq } from './YAMLSeq.js'
 
-export type Node = Alias | Scalar | YAMLMap | YAMLSeq
+export type Node<T = unknown> =
```

Comment:
May be it would make more sense to rename `K` to `Key` and both `T` and `V` to `Value` across the whole codebase? Because now the fact that `T` and `V` have the same meaning (type of the value) may not be immediately obvious for the reader.

Reply:
Yeah, probably. The generic type identifiers are a bit terse.